lib/spack/spack/cmd/unit_test.py

```diff
@@ -214,8 +214,6 @@ def unit_test(parser, args, unknown_args):
 
     # Ensure clingo is available before switching to the
     # mock configuration used by unit tests
-    # Note: skip on windows here because for the moment,
-    # clingo is wholly unsupported from bootstrap
```

Comment:
(no change needed) I'm assuming this is just removing a comment that was outdated before this PR started.

Reply:
Correct, just happened to notice it while debugging that previous spurious issue.